apps/src/templates/sectionAssessments/MultipleChoiceAssessmentsOverviewContainer.jsx

```diff
@@ -53,6 +52,6 @@ export const UnconnectedMultipleChoiceAssessmentsOverviewContainer = MultipleCho
 export default connect(state => ({
   questionAnswerData: getMultipleChoiceSectionSummary(state),
   totalStudentSubmissions: countSubmissionsForCurrentAssessment(state),
-  totalStudentCount: getTotalStudentCount(state),
+  totalStudentCount: state.teacherSections.selectedStudents.length,
```

Comment:
Is state.teacherSections.selectedStudents guaranteed to be non-null?

Reply:
It should be an empty array, I just tested with a section with no students and it works. We could probably be more intentional about guarding against null here, I'll take a look and see if there's anything worth adding right now.